rir/src/interpreter/interp.c

```diff
@@ -97,7 +94,6 @@ INLINE SEXP createPromise(Code* code, SEXP env) {
     SET_ATTRIB(p, a);
```

Comment:
I think this hack is (a) broken and (b) not needed. We should remove it as well. But I can look into it.